demo_sinatra/app.rb

```diff
@@ -554,7 +554,7 @@ def highchart_stock_map
     }
   ]
 
-  @area_range = Daru::View::Plot.new(series_dt6, opts6)
+  @area_range = Daru::View::Plot.new(series_dt6, opts6, user_options)
 
   opts8 = {
```

Comment:
Very disappointed to see variable names  :(

Reply:
Really sorry about this. As discussed, I just added the examples that I was using locally to check the features. I will fix this soon.